libImaging/Resample.c

```diff
@@ -107,7 +107,7 @@ ImagingPrecompute(int inSize, int outSize, struct filter *filterp,
     if ( ! kk)
         return 0;
 
-    xbounds = calloc(outSize * 2, sizeof(int));
+    xbounds = malloc(outSize * 2 * sizeof(int));
```

Comment:
What exactly do you mean? ;)
`calloc` is just a function and it does what it should.

I still don't understand why calloc failed on windows while malloc didn't. In my opinion it should depend on pointer size and SIZE_MAX check was correct while the test with fixed size wasn't. But I can't waste much time to investigate real reasons especially when I don't have windows PC. The INT_MAX fix perfectly fits me.